pkg/relayer/config/types.go

```diff
@@ -179,7 +202,7 @@ type RelayMinerSupplierConfig struct {
 // RelayMinerSupplierServiceConfig is the structure resulting from parsing the supplier
 // service sub-section of the RelayMiner config file.
 type RelayMinerSupplierServiceConfig struct {
-	// BackendUrl is the URL of the service that relays will be proxied to.
+	// BackendUrl is the default URL of the service that relays will be proxied to.
```

Comment:
Why `default`. Shouldn't it be the "only` URL given a `RelayMinerSupplierServiceConfig` ?